eZ/Publish/Core/Repository/LocationResolver/PermissionAwareLocationResolver.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace eZ\Publish\Core\Repository\LocationResolver;
+
+use eZ\Publish\API\Repository\Exceptions\NotFoundException;
+use eZ\Publish\API\Repository\Exceptions\UnauthorizedException;
+use eZ\Publish\API\Repository\LocationService;
+use eZ\Publish\API\Repository\Values\Content\ContentInfo;
+use eZ\Publish\API\Repository\Values\Content\Location;
+use eZ\Publish\Core\Base\Exceptions\NotFoundException as NotFoundExceptionCore;
```

Comment:
Minor optional nitpick: usually following the NS order: `as CoreNotFoundException`.

Reply:
Fixed in https://github.com/ezsystems/ezpublish-kernel/pull/3050/commits/52c321a8aaeb1959597da4637a6d8e58256d286c